### PR TITLE
Update Ubuntu install instructions to use install script

### PR DESCRIPTION
**Pull request by mheap**
### Description

Initial attempt at a refactored Ubuntu install page.

The script has been tested on Ubuntu 22.04 and 20.04 (both LTS) on AWS, and on a local docker container.

I've intentionally removed the include as we have dedicated pages that I've linked to in next steps

### Testing instructions

Netlify link: https://deploy-preview-5389--kongdocs.netlify.app/gateway/3.2.x/install/linux/ubuntu/

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_src/gateway/install/linux/ubuntu.md [hunk 1]
```diff
@@ -7,12 +7,62 @@ The {{site.base_gateway}} software is governed by the
...
+
+```bash
+curl -i http://localhost:8001
+```
+
+You should receive a `200` status code.
+
+### Next steps
+
+Once Kong Gateway is running, you may want to:
```
Comment (mheap) [inline app/_src/gateway/install/linux/ubuntu.md:56]:
```suggestion
Once {{site.base_gateway}} is running, you may want to:
```

File: app/_src/gateway/install/linux/ubuntu.md [hunk 2]
```diff
@@ -7,12 +7,62 @@ The {{site.base_gateway}} software is governed by the
...
+{% navtab Kong Gateway (OSS) %}
+```bash
+bash <(curl -sS https://install-script--getkong.netlify.app/install) -p kong -v {{page.versions.ce}}
+```
+{% endnavtab %}
+{% endnavtabs_ee %}
+
+This script will detect your operating system and automatically install the correct package. It'll also install a Postgres database and bootstrap {{ site.base_gateway }} for you.
+
+If you'd prefer to install just the package, please see the [Package Install](#package-install) section.
```
Comment (cloudjumpercat) [inline app/_src/gateway/install/linux/ubuntu.md:42]: I think it's just my lack of understanding, but why would you want to just install the package? Or maybe conversely, why would you want to install Gateway, Postgres, and bootstrap Gateway at the same time with the script? But maybe the target audience would already know the answer to this and wouldn't need it spelled out.

File: app/_src/gateway/install/linux/ubuntu.md [hunk 3]
```diff
@@ -7,12 +7,62 @@ The {{site.base_gateway}} software is governed by the
...
+
+```bash
+curl -i http://localhost:8001
+```
+
+You should receive a `200` status code.
+
+### Next steps
+
+Once {{ site.base_gateway }} is running, you may want to:
```
Comment (cloudjumpercat) [inline app/_src/gateway/install/linux/ubuntu.md:56]:
```suggestion
Once {{ site.base_gateway }} is running, you may want to do the following:
```

File: app/_src/gateway/install/linux/ubuntu.md [hunk 4]
```diff
@@ -7,12 +7,62 @@ The {{site.base_gateway}} software is governed by the
...
+
+* [Add your Enterprise license (optional)](/gateway/{{ page.kong_version }}/licenses/deploy).
```
Comment (cloudjumpercat) [inline app/_src/gateway/install/linux/ubuntu.md:58]:
```suggestion
* Optional: [Add your Enterprise license](/gateway/{{ page.kong_version }}/licenses/deploy).
```

File: app/_src/gateway/install/linux/ubuntu.md [hunk 5]
```diff
@@ -129,9 +161,22 @@ apt install -y kong={{page.versions.ce}}
 
 {{ install_from_repo | indent | replace: " </code>", "</code>" }}
 
+4. (Optional) Prevent accidental upgrades by marking the package as `hold`:
```
Comment (cloudjumpercat) [inline app/_src/gateway/install/linux/ubuntu.md:164]:
```suggestion
4. Optional: Prevent accidental upgrades by marking the package as `hold`:
```

**Review by cloudjumpercat — COMMENTED** [review]
I didn't test out the scripts, just read the page over. Overall, I like the simplicity of it and that we provide the easiest option first (but still give an advanced option if people want it). I had one question about adding more info about who would want to use the advanced option instead of the easy option, but we may not need to add that.

I'll wait to approve in case @Guaris and @lena-larionova want to look it over.

**Comment by acgoldsmith** [comment]
Emphatic +1 to @cloudjumpercat 's praise of the restructure into easier and advanced. That's a big improvement and great pattern to follow for other install topics.

File: app/_src/gateway/install/linux/ubuntu.md [hunk 6]
```diff
@@ -7,12 +7,62 @@ The {{site.base_gateway}} software is governed by the
...
 
 * A supported system with root or [root-equivalent](/gateway/{{page.kong_version}}/production/running-kong/kong-user/) access.
 * (Enterprise only) A `license.json` file from Kong
 
-## Download and install
+## Installation
+
+The quickest way to get started with {{ site.base_gateway }} is using our install script:
+
+{% navtabs_ee codeblock %}
```
[the remark this answers is not in the document]
Comment (mheap) [reply app/_src/gateway/install/linux/ubuntu.md:27]: The nav tabs don't exist when using `navtabs_ee` as they're controlled by this toggle

CleanShot 2023-04-21 at 12 07 37

File: app/_src/gateway/install/linux/ubuntu.md (hunk 2 above)
Comment (mheap) [reply app/_src/gateway/install/linux/ubuntu.md:42]: This comes down to the target audience for me. Either:

* They know why they need a package only and have the skills required to manage a package install + external DB
* They don't know why they need a package only install, and the install script is what they should use

If we get more questions on this we can expand the _why_ but for now I'd like to keep the page streamlined

**Review by Guaris — APPROVED** [review]

**Pull request merged**